administrative-sdk/choice-recognition/choice-recognition-controller.js

```diff
@@ -55,6 +56,7 @@ module.exports = class ChoiceRecognitionController {
       // RPC error callback
       res => {
         console.error('RPC error returned:', res.error);
+        throw res;
```

Comment:
Errors shouldn't be thrown in promises. Instead `return Promise.reject(res)`.

This goes for all occurrences.